.ado/templates/react-native-init.yml

```diff
@@ -102,7 +102,6 @@ steps:
       msbuildArgs:
         /p:PreferredToolArchitecture=$(MSBuildPreferredToolArchitecture)
         /p:PlatformToolset=$(MSBuildPlatformToolset)
-        /p:BaseIntDir=$(BaseIntDir)
         /p:AppxGeneratePrisForPortableLibrariesEnabled=false
```

Comment:
@jonthysell Do we need this specialization here?

Reply:
Ultimately, I think we should switch the react-native init task to use run-windows, since that's exactly what customers will use. https://github.com/microsoft/react-native-windows/issues/4464

We can pass options if we want, but we should really be testing that the cli works, not that we can run the cli once and then build with VS.

Reply:
Conversely, SampleApps is using run-windows, and that can be changed into a VSbuild task